cfgrib/dataset.py

```diff
@@ -233,6 +233,7 @@
         "units": "hours",
         "standard_name": "forecast_period",
         "long_name": "time since forecast_reference_time",
+        "dtype": "timedelta64[s]",
```

Comment:
```suggestion
        "dtype": "timedelta64[ns]",
```
It looks like old XArray versions do read this parameter, and object if it's not one they support.